app/controllers/admin/users_controller.rb

```diff
@@ -18,15 +18,26 @@ def dashboard
 
   def update
     user = User.find(params[:user_id])
+
```

Comment:
because we have enums on users we should be able to call user.merchant? insted of doing the user.role == merchant stuff

Reply:
Changes made, good catch Ty.